content/integrations/statsd.md

```diff
@@ -7,12 +7,9 @@ git_integration_title: statsd
 
 # Overview
 
-Enable the Datadog StatsD integration to:
+This Agent check monitors the availability and uptime of non-Datadog StatsD servers. It also tracks the rate StatsD servers are receiving metrics.
```

Comment:
Clean and nice! Maybe end with "receiving application metrics", though maybe "application" qualifier is unnecessary.